src/ruby/yast/wfm.rb

```diff
@@ -308,6 +308,11 @@ def self.call_builtin_wrapper(*args)
         Yast.import "Report"
         Report.LongError(msg, height: height)
       end
+
+      # Logging all information about the product evaluation
+      Yast.import "ProductEvaluation"
+      ProductEvaluation.write("internal_error")
```

Comment:
Maybe we should also write some message into the file describing the problem:
```suggestion
      ProductEvaluation.write("internal_error", { error: "Exception raised in #{client}",
        error_details: msg, error_backtrace: e.backtrace })
```

Just simply merge the optional extra keys passed to the function.